bot/SciencePlayer.py

```diff
@@ -0,0 +1,56 @@
+from typing import Dict
+
+from bot.BasePlayer import BasePlayer
+from util.cardUtils import get_all_cards_dict
+from util.constants import SCIENCE
+
+
+class SciencePlayer(BasePlayer):
+    def __init__(self):
+        super().__init__()
+        self.all_cards = get_all_cards_dict()
+
+    def _get_lowest_cost_card_id(self, cards: dict):
+        lowest_id = ""
+        payment_option = 0
+        payment_cost = 999  # big number to start
+        for card_id, cost in cards.items():
+            min_cost = min((sum(payment) for payment in cost), default=999)
+            if min_cost < payment_cost:
+                for i in range(len(cost)):
+                    if sum(cost[i]) < payment_option:
+                        payment_option = i
+                payment_cost = min_cost
+                lowest_id = card_id
+        return lowest_id
```

Comment:
Functions that return values should have types defined in the function signature